catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/FrameworkProperties.java

```diff
@@ -81,7 +81,7 @@
 
   private List<ConnectedSource> connectedSources = new ArrayList<>();
 
-  private Map<String, FederatedSource> federatedSources = new HashMap<>();
+  private Collection<FederatedSource> federatedSources = Collections.emptyList();
```

Comment:
❓ Should we use a thread-safe collection here?

Reply:
see [above](https://github.com/codice/ddf/pull/4250#discussion_r256467716)